Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -171,7 +171,11 @@ def compiler
 
   # @private
   def determine_cc
-    COMPILER_SYMBOL_MAP.invert.fetch(compiler, compiler)
+    if compiler == :brew_clang
+      "/usr/local/opt/llvm/bin/clang"
```

Comment:
Is there an environment variable version of `brew --prefix llvm`? Or should I just use the prefix + `opt/llvm`?